seedVault/index.ts

```diff
@@ -49,9 +49,13 @@ class SeedVault {
     this._secureStorageAdapter.set(SeedVaultStorageKeys.PASSWORD_HASH, passwordHash);
   };
 
-  storeSeed = async (seed: string) => {
+  storeSeed = async (seed: string, overwriteExistingSeed?: boolean) => {
     const password = await this._secureStorageAdapter.get(SeedVaultStorageKeys.PASSWORD_HASH);
     if (!password) throw new Error('passwordHash not set');
+    const prevStoredEncryptedSeed = await this._commonStorageAdapter.get(SeedVaultStorageKeys.ENCRYPTED_KEY);
+    if (prevStoredEncryptedSeed && prevStoredEncryptedSeed !== '' && !overwriteExistingSeed) {
```

Comment:
Empty strings are falsey by default, no need for the extra check.

```suggestion
    if (prevStoredEncryptedSeed && !overwriteExistingSeed) {
```